main.cpp

```diff
@@ -79,8 +379,35 @@ If you need inspiration for what to write, take a look at previously approved st
  Wait for my code review.
  */
 
-#include <iostream>
+
 int main()
 {
+    Wizard Fizban;
+    Fizban.castSpell("boof!");
+    Fizban.runAway(30);
+    Fizban.hide(false);
+    cout << "Haha! Fizban of " << Fizban.age << " years boofed you!" << endl;
+    Fighter Mooky;
+    Mooky.fight("mace");
+    Mooky.block(66);
+    Mooky.drink(11);
+    cout << "Haha! Mooky with the signet " << Mooky.signet << " gets dragged off by gnomes" << endl;
+    Preistess Pearl;
+    Pearl.castGrowSpell(7);
+    Pearl.castBanish("Bad Smells");
+    Pearl.castIllusions(7);
+    Thief Smarkles;
+    Smarkles.cartwheel(13);
+    Smarkles.smoothTalk();
+    Smarkles.steal(100.00f);
+    Bard Fabio;
+    Fabio.beautyRest();
+    Fabio.playMusic();
+    Fabio.jump(5.0f);
+    Wizard Barth;
+    Fighter Marmy;
+    Preistess periwinkle;
+    Thief sammy;
+    Bard Ahsia;
```

Comment:
Type names begin with a Capital letter
variable names begin with a lowercase letter
correct the capitalization of all of these variables
- [ ] revise